Celeste.Mod.mm/Mod/Entities/EntityCollider.cs

```diff
@@ -35,6 +54,10 @@ public override void Added(Entity entity) {
                 if (!Scene.Tracker.IsEntityTracked<T>()) {
                     patch_Tracker.AddTypeToTracker(typeof(T));
                 }
+                if (!Scene.Tracker.IsComponentTracked<EntityCollider<T>>()) {
+                    patch_Tracker.AddTypeToTracker(typeof(EntityCollider<T>));
+                    patch_Tracker.AddTypeToTracker(typeof(EntityCollider<T>), typeof(EntityCollider));
+                }
```

Comment:
Add Type to tracker should be formatted (superclass, subclass) so only one line is needed with the superclass EntityCollider being first

Reply:
Unless this is the TrackedAs type?

Reply:
if that's the case then why do you need to track them normally and TrackedAs at the same time?

Reply:
yes, this is [tracked] and [trackedas]
then you are able to get all EntityCollider of specific type and all EntityCollider of unknown types

Reply:
but [tracked] may not be good, we can require modders to track them

Reply:
I don't see the point of even needing these Colliders in the Tracker now that I think about it. What purpose would you even need to track them with